app/encoding/index_wrapper_decoder.go

```diff
@@ -5,10 +5,10 @@ import (
 	coretypes "github.com/tendermint/tendermint/types"
 )
 
-func WrappedTxDecoder(dec sdk.TxDecoder) sdk.TxDecoder {
+func IndexWrapperDecoder(dec sdk.TxDecoder) sdk.TxDecoder {
```

Comment:
[super minor no blocking comment]
I know no one uses this function, but does this mean this is technically API breaking? (so chore! ?)

Reply:
oof yea it's technically exported public API so this should've been `chore!`